bitnami/keycloak/templates/servicemonitor.yaml

```diff
@@ -1,13 +1,14 @@
-{{- if and .Values.metrics.enabled .Values.metrics.serviceMonitor.enabled }}
+{{- $serviceMonitor := .Values.metrics.serviceMonitor }}
```

Comment:
I see that you are using intermediate variables like these. I imagine you are using them for simplicity, right? The issue is that it differs from the other charts and it would make it a bit more difficult to us to apply batch changes in all charts. Could you remove these intermediate vars?

Reply:
Yep, sure. Just some "muscle memory" to do that. But I see your point. Removed them from both the ServiceMonitor and the PrometheusRule templates.